README.md

```diff
@@ -31,15 +40,20 @@ _--image-name-match_ cannot match the replacement image.
 zypper in python3-ec2imgutils
 ```
 
+
 ## Usage
 
 ```
 ec2deprecateimg --account example --image-name-match v15 --image-virt-type hvm --replacement-name exampleimage_v16
+
+ec2publishimg --account example --image-name-match production-v2 --share-with all
```

Comment:
I think it'd be nice to organize each sub-utility into it's own readme section. E.g.

```
# Overall
## Requirements
## Installation
## Provides
## ec2deprecateimg

Info...

### Usage
### Man
## ec2publishimg

Info...

### Usage
### Man
```

Reply:
We'll do README refactor once everything is merged.

Reply:
:+1: